package.json

```diff
@@ -2,10 +2,12 @@
   "name": "@wecount/wecount.inclusivedesign.ca",
   "version": "1.0.1",
   "description": "The source files for the We Count website, built with Nuxt.",
-  "license": "CC-BY-4.0",
+	"license": "CC-BY-4.0",
+	"author": "OCAD University <idrc@ocadu.ca>",
```

Comment:
@greatislander I think this is the last thing, there are mixed tabs and spaces in this file.